infinispan-embedded-query and infinispan-remote-query modules broken

Broken in jetty-10 and jetty-11 branches.

> 
> 2021-09-09 10:52:17.777:WARN :oejx.XmlConfiguration:main: Config error java.lang.ClassNotFoundException: org.eclipse.jetty.server.session.infinispan.InfinispanSessionData at <Call name="entity"><Arg>|              <Get class="org.eclipse.jetty.server.session.infinispan.InfinispanSessionData" name="class"/>|            </Arg><Call name="indexed"><Call name="providedId"><Call name="property"><Arg type="String">expiry</Arg><Arg>|                    <Get class="java.lang.annotation.ElementType" name="FIELD"/>|                  </Arg><Call name="field"/></Call></Call></Call></Call> in file:///home/janb/src/jetty-eclipse/jetty-10.0.x/jetty-home/target/jetty-home/etc/sessions/infinispan/infinispan-embedded-query.xml
> 2021-09-09 10:52:17.777:WARN :oejx.XmlConfiguration:main: Config error java.lang.ClassNotFoundException: org.eclipse.jetty.server.session.infinispan.InfinispanSessionData at <Call name="put"><Arg>|        <Get class="org.hibernate.search.cfg.Environment" name="MODEL_MAPPING"/>|      </Arg><Arg>|        <New class="org.hibernate.search.cfg.SearchMapping"><Call name="entity"><Arg>|              <Get class="org.eclipse.jetty.server.session.infinispan.InfinispanSessionData" name="class"/>|            </Arg><Call name="indexed"><Call name="providedId"><Call name="property"><Arg type="String">expiry</Arg><Arg>|                    <Get class="java.lang.annotation.ElementType" name="FIELD"/>|                  </Arg><Call name="field"/></Call></Call></Call></Call></New>|      </Arg></Call> in file:///home/janb/src/jetty-eclipse/jetty-10.0.x/jetty-home/target/jetty-home/etc/sessions/infinispan/infinispan-embedded-query.xml
> 2021-09-09 10:52:17.778:WARN :oejx.XmlConfiguration:main: Config error java.lang.ClassNotFoundException: org.eclipse.jetty.server.session.infinispan.InfinispanSessionData at <New id="properties" class="java.util.Properties"><Call name="put"><Arg>|        <Get class="org.hibernate.search.cfg.Environment" name="MODEL_MAPPING"/>|      </Arg><Arg>|        <New class="org.hibernate.search.cfg.SearchMapping"><Call name="entity"><Arg>|              <Get class="org.eclipse.jetty.server.session.infinispan.InfinispanSessionData" name="class"/>|            </Arg><Call name="indexed"><Call name="providedId"><Call name="property"><Arg type="String">expiry</Arg><Arg>|                    <Get class="java.lang.annotation.ElementType" name="FIELD"/>|                  </Arg><Call name="field"/></Call></Call></Call></Call></New>|      </Arg></Call></New> in file:///home/janb/src/jetty-eclipse/jetty-10.0.x/jetty-home/target/jetty-home/etc/sessions/infinispan/infinispan-embedded-query.xml
> 2021-09-09 10:52:17.778:WARN :oejx.XmlConfiguration:main: Unable to execute XmlConfiguration
> java.security.PrivilegedActionException: java.lang.ClassNotFoundException: org.eclipse.jetty.server.session.infinispan.InfinispanSessionData
> 	at java.base/java.security.AccessController.doPrivileged(Native Method)
> 	at org.eclipse.jetty.xml.XmlConfiguration.main(XmlConfiguration.java:1810)
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> 	at org.eclipse.jetty.start.Main.invokeMain(Main.java:227)
> 	at org.eclipse.jetty.start.Main.start(Main.java:512)
> 	at org.eclipse.jetty.start.Main.main(Main.java:74)

> Caused by: 
> java.lang.ClassNotFoundException: org.eclipse.jetty.server.session.infinispan.InfinispanSessionData
> 	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:471)
> 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
> 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
> 	at org.eclipse.jetty.util.Loader.loadClass(Loader.java:59)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.get(XmlConfiguration.java:833)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.itemValue(XmlConfiguration.java:1491)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.value(XmlConfiguration.java:1394)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration$Args.<init>(XmlConfiguration.java:1652)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.call(XmlConfiguration.java:928)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.configure(XmlConfiguration.java:464)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.newObj(XmlConfiguration.java:1008)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.itemValue(XmlConfiguration.java:1493)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.value(XmlConfiguration.java:1394)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration$Args.<init>(XmlConfiguration.java:1652)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.call(XmlConfiguration.java:928)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.configure(XmlConfiguration.java:464)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.newObj(XmlConfiguration.java:1008)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.configure(XmlConfiguration.java:470)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.configure(XmlConfiguration.java:417)
> 	at org.eclipse.jetty.xml.XmlConfiguration.configure(XmlConfiguration.java:319)
> 	at org.eclipse.jetty.xml.XmlConfiguration.lambda$main$4(XmlConfiguration.java:1848)
> 	at java.base/java.security.AccessController.doPrivileged(Native Method)
> 	at org.eclipse.jetty.xml.XmlConfiguration.main(XmlConfiguration.java:1810)
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> 	at org.eclipse.jetty.start.Main.invokeMain(Main.java:227)
> 	at org.eclipse.jetty.start.Main.start(Main.java:512)
> 	at org.eclipse.jetty.start.Main.main(Main.java:74)
> java.lang.reflect.InvocationTargetException
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> 	at org.eclipse.jetty.start.Main.invokeMain(Main.java:227)
> 	at org.eclipse.jetty.start.Main.start(Main.java:512)
> 	at org.eclipse.jetty.start.Main.main(Main.java:74)
> Caused by: java.security.PrivilegedActionException: java.lang.ClassNotFoundException: org.eclipse.jetty.server.session.infinispan.InfinispanSessionData
> 	at java.base/java.security.AccessController.doPrivileged(Native Method)
> 	at org.eclipse.jetty.xml.XmlConfiguration.main(XmlConfiguration.java:1810)
> 	... 7 more
> Caused by: java.lang.ClassNotFoundException: org.eclipse.jetty.server.session.infinispan.InfinispanSessionData
> 	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:471)
> 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
> 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
> 	at org.eclipse.jetty.util.Loader.loadClass(Loader.java:59)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.get(XmlConfiguration.java:833)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.itemValue(XmlConfiguration.java:1491)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.value(XmlConfiguration.java:1394)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration$Args.<init>(XmlConfiguration.java:1652)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.call(XmlConfiguration.java:928)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.configure(XmlConfiguration.java:464)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.newObj(XmlConfiguration.java:1008)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.itemValue(XmlConfiguration.java:1493)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.value(XmlConfiguration.java:1394)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration$Args.<init>(XmlConfiguration.java:1652)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.call(XmlConfiguration.java:928)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.configure(XmlConfiguration.java:464)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.newObj(XmlConfiguration.java:1008)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.configure(XmlConfiguration.java:470)
> 	at org.eclipse.jetty.xml.XmlConfiguration$JettyXmlConfiguration.configure(XmlConfiguration.java:417)
> 	at org.eclipse.jetty.xml.XmlConfiguration.configure(XmlConfiguration.java:319)
> 	at org.eclipse.jetty.xml.XmlConfiguration.lambda$main$4(XmlConfiguration.java:1848)

--- a/jetty-infinispan/infinispan-embedded-query/src/main/config-template/etc/sessions/infinispan/infinispan-embedded-query.xml
+++ b/jetty-infinispan/infinispan-embedded-query/src/main/config-template/etc/sessions/infinispan/infinispan-embedded-query.xml
@@ -18,7 +18,7 @@
         <New class="org.hibernate.search.cfg.SearchMapping">
           <Call name="entity">
             <Arg>
-              <Get class="org.eclipse.jetty.server.session.infinispan.InfinispanSessionData" name="class"/>
+              <Get class="org.eclipse.jetty.session.infinispan.InfinispanSessionData" name="class"/>
             </Arg>
             <Call name="indexed">
               <Call name="providedId">
@@ -57,7 +57,7 @@
           </Arg>
           <Call name="addIndexedEntity">
             <Arg>
-              <Get class="org.eclipse.jetty.server.session.infinispan.InfinispanSessionData" name="class"/>
+              <Get class="org.eclipse.jetty.session.infinispan.InfinispanSessionData" name="class"/>
             </Arg>
             <Call name="withProperties">
               <Arg>

--- a/jetty-infinispan/infinispan-remote-query/src/main/config-template/etc/sessions/infinispan/infinispan-remote-query.xml
+++ b/jetty-infinispan/infinispan-remote-query/src/main/config-template/etc/sessions/infinispan/infinispan-remote-query.xml
@@ -8,7 +8,7 @@
   <!-- ===================================================================== -->
   <New id="mapping" class="org.hibernate.search.cfg.SearchMapping">
     <Call name="entity">
-      <Arg><Call class="java.lang.Class" name="forName"><Arg>org.eclipse.jetty.server.session.infinispan.InfinispanSessionData</Arg></Call></Arg>
+      <Arg><Call class="java.lang.Class" name="forName"><Arg>org.eclipse.jetty.session.infinispan.InfinispanSessionData</Arg></Call></Arg>
       <Call name="indexed">
         <Call name="providedId">
           <Call name="property">

--- a/tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/session/AbstractSessionDistributionTests.java
+++ b/tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/session/AbstractSessionDistributionTests.java
@@ -100,12 +100,14 @@ public abstract class AbstractSessionDistributionTests extends AbstractJettyHome
                 assertThat(response.getContentAsString(), containsString("SESSION READ CHOCOLATE THE BEST:FRENCH"));
             }
 
+            /*
             Path logFile = jettyHomeTester.getJettyBase().resolve("resources").resolve("jetty-logging.properties");
             Files.deleteIfExists(logFile);
             try (BufferedWriter writer = Files.newBufferedWriter(logFile, StandardCharsets.UTF_8, StandardOpenOption.CREATE))
             {
                 writer.write("org.eclipse.jetty.server.session.LEVEL=DEBUG");
             }
+            */
 
             try (JettyHomeTester.Run run2 = jettyHomeTester.start(argsStart))
             {

--- a/tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/session/InfinispanSessionDistributionTests.java
+++ b/tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/session/InfinispanSessionDistributionTests.java
@@ -104,7 +104,7 @@ public class InfinispanSessionDistributionTests extends AbstractSessionDistribut
     @Override
     public String getFirstStartExtraModules()
     {
-        return "session-store-infinispan-remote";
+        return "session-store-infinispan-remote,infinispan-remote-query";
     }
 
     @Override

